src/test/tools/bzlmod/MODULE.bazel.lock

```diff
@@ -150,11 +151,11 @@
           "pybind11": {
             "repoRuleId": "@@bazel_tools//tools/build_defs/repo:http.bzl%http_archive",
             "attributes": {
-              "build_file": "@@pybind11_bazel+//:pybind11.BUILD",
```

Comment:
Do you know why all these attributes are now ordered differently? It seems safer to preserve the original order.

Reply:
the reason is that, when we produce a `RepoSpec` from a repo rule instantiation, we now sort the attr values by attr schema order, instead of params-at-rule-invocation-time order. And this order stays with the `RepoSpec` object (which means that the `bazel mod show_repo` thing also retains this order).

in hindsight, we should probably use the old sort order. `bazel mod show_repo` returning a different order is probably much less disruptive than the lockfile shuffling... Will update.